src/main/scala/com/datastax/sparkstress/ReadTask.scala

```diff
@@ -53,7 +68,21 @@ abstract class ReadTask(config: Config, sc: SparkContext) extends StressTask {
 class PDCount(config: Config, sc: SparkContext) extends ReadTask(config, sc) {
 
   def run(): Unit = {
-    val count = sc.cassandraTable(keyspace, table).cassandraCount()
+    val count = sc.cassandraTable("ks","tab").cassandraCount()
+    if (config.totalOps != count) {
+      println(s"Read verification failed! Expected ${config.totalOps}, returned $count");
+    }
+    println(count)
+  }
+}
+
+/**
+  * SparkSQL Count
+  */
+class SparkSqlCount(config: Config, sc: SparkContext) extends ReadTask(config, sc) {
+
+  def run(): Unit = {
+    val count = sqlContext.sql("SELECT COUNT(*) FROM ks.tab").toDF("count").first.getAs[Long]("count")
     if (config.totalOps != count) {
```

Comment:
Well in the other read tests we would have to know a priori what the expected number of returned rows would be. This is the same case here obviously, but here at least it's a bit more tractable, the user just needs to make sure they specify the same `totalOps` number used for the prerequisite write workload.